src/openrct2-ui/CursorData.cpp

```diff
@@ -46,7 +46,7 @@ namespace OpenRCT2::Ui
 
     static constexpr const CursorData UpDownArrowCursorData =
     {
-        { 7, 31 },
+        { 17, 16 },
```

Comment:
Just revert all changes except for this line. I tested this change, and it looks good.

Reply:
You could reopen this one and make the required changes, but I see you already opened a new one instead.

Reply:
I think non-members can't reopen PRs. If you have trouble doing that, you can open a new one